src/tqec/compile/compile_test.py

```diff
@@ -59,26 +36,32 @@ def test_compile_single_block_memory(spec: str, kind: str, k: int) -> None:
 
 
 @pytest.mark.parametrize(
-    ("spec", "kind", "k"),
-    itertools.product(SPECS.keys(), ("ZXZ", "ZXX", "XZX", "XZZ"), (1,)),
+    ("spec", "kind", "k", "xy"),
+    itertools.product(
+        ALL_SPECS.keys(),
+        ("ZXZ", "ZXX", "XZX", "XZZ"),
+        (1,),
+        ((0, 0), (1, 1), (2, 2)),
```

Comment:
Would be nice to have: `(0, 0)`, some positive numbers (such as `(1, 1)`) and some negative numbers.